scripts/specs.ts

```diff
@@ -1,18 +1,54 @@
+import assert from "node:assert/strict";
+
 import webSpecs from 'web-specs' assert { type: 'json' };
 
 import features from '../index.js';
 
-const specUrls = new Set(webSpecs.map(spec => spec.nightly.url));
+const specUrls = webSpecs.map(spec => new URL(spec.nightly.url));
+
+type exception = [url: string, message: string];
```

Comment:
I suggest avoiding the word exception as it seems like something that would be thrown.

```suggestion
type allowlistItem = [url: string, message: string];
```